src/components/Admin/UserPage/ManageUsers.tsx

```diff
@@ -31,6 +92,45 @@ const ManageUsers: () => JSX.Element = () => {
           textAlign="center"
         >
           <Heading>Manage Users</Heading>
+          <Table>
+            <TableCaption placement="top">Current Admin Users</TableCaption>
+            <Thead>
+              <Tr>
+                <Th>Name</Th>
+                <Th>Email</Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {users
+                .filter(user => user.admin)
+                .map((user, id) => (
+                  <Tr key={id}>
+                    <Td>{user.name}</Td>
+                    <Td>{user.email}</Td>
+                  </Tr>
+                ))}
+            </Tbody>
+          </Table>
+          <Table>
+            <TableCaption placement="top">All Current Users</TableCaption>
```

Comment:
This caption is centered differently than "Current Admin Users", but I am not sure why... (it's closer to the right side of the screen)

Reply:
I changed this to a heading so that we can control the styling more directly